python/adelphi/adelphi/export.py

```diff
@@ -95,7 +95,8 @@ def make_tuple(ks):
             if props['anonymize']:
                 anonymize_keyspace(ks)
             return KsTuple(ids[orig_name], ks)
-        return {t.ks_obj.name : t for t in [make_tuple(ks) for ks in keyspaces]}
+        tuples = sorted([make_tuple(ks) for ks in keyspaces], key=lambda ks: ks.ks_obj.name)
+        return OrderedDict([(t.ks_obj.name,t) for t in tuples])
```

Comment:
Keyspaces returned via export should be handled in an orderly way.  This is essential for legit compares of prior output to what's done by the current code, which in turn is an essential feature for our integration tests.